### PR TITLE
Porfolio

### PR DESCRIPTION
**Pull request by LethaboMot**
in this project I:

-added html
-added css
-added readme file

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @LethaboMot  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid


## Invalid Code Review Request

You have submitted a project with the readme file missing certain key headers/sections like the `key features` and `future features`. Kindly ensure that all sections/headers of the readme file is present and updated.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  gap: 12px;
+}
+
+#Introduction h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 60px;
+  color: #253652;
+  padding: 0 24px;
+}
```
Comment (mckent05) [inline style.css:53]: - Please reduce the font size used here for the `h1` tag, so your design can look more like the figma design template. The recommended font size is `40px`. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this 👍 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    <title>Document</title>
+</head>
+
+<body>
+    <header id="toppart">
+        <nav class="toolbar">
+            <a href="">
+                <h2><img src="images/My Logo.svg" width="90" height="90"></h2>
+            </a>
```
Comment (mckent05) [inline index.html:18]: - For semantic reasons, it is not advisable to wrap the `img` tag in an `h2` tag, I suggest you wrap the `img` tag directly under the `a` tag. Please fix this 🙏 🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                <a href="#"><img src="images/Medium.svg" alt="Medium"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/GitHub.svg" alt="GitHub"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Angel list.svg" alt="ANGEL LIST"></a>
+            </li>
+        </ul>
+    </section>
```
Comment (mckent05) [inline index.html:48]: - Good job you have done so far 👏, but there are some little differences between your site and the design template. Please notice that at the bottom left corner of the design template, there is a border-radius. Please fix this in your design. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
tzetjetj

Figma template:
egnengn

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @LethaboMot,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ 
- Your project looks good 👏 👏 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @LethaboMot , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: images/index.html [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+            <a href=""><img src="images/Hamburger button.svg" alt="three lines" width="24" height="24"></a>
+        </nav>
+    </header>
+
+    <main class="main">
+        <section id="Introduction">
+            <h1>
+                <p>I'm Lethabo</p>
+                <p>Glad to see you!</p>
+            </h1>
```
[the remark this answers is not in the document]
Comment (LethaboMot) [reply images/index.html:28]: I didn't put a <p> tag in  my heading though

[the remark this answers is not in the document]
Comment (LethaboMot) [reply images/index.html:28]: I meant to say I didn't add a   "p" tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                <img src="images/My Logo.svg" width="90" height="90">
+            </a>
+            <a href=""><img src="images/Hamburger button.svg" alt="three lines" width="24" height="24"></a>
+        </nav>
+    </header>
+
+    <section id="Introduction">
+        <h1>I'm Lethabo Glad to see you!</h1>
+        <div class="mid">
+            <p style="font-size: 4.3vw;">I'm a software developer! I can help you build a product, feature or website Look through some fo my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
```
Comment (Yerimah) [inline index.html:26]: - [ ] I would suggest you maintain a specific format for styling your HTML document to comply with HTML/CSS best practices. All your style definitions must be kept in a separate CSS file. 😊

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES 🔴

Hi @LethaboMot 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

So Close Snl GIF by Saturday Night Live

 ## Highlights
✅ All linter checks are passing
✅ Your project looks good 👏 👏
✅ Your PR is professional 👍 👍


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (LethaboMot) [reply index.html:26]: what does this mean

Comment (LethaboMot) [reply index.html:26]: nevermind

Comment (LethaboMot) [reply index.html:26]: i understand the issue

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @ndorrh , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] [OPTIONAL] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Pull request merged**